test/client.js

```diff
@@ -586,3 +586,35 @@ test('close should call callback once finished', (t) => {
     }
   })
 })
+
+test('close should still reconnect', (t) => {
+  t.plan(6)
+
+  const server = createServer((req, res) => {
+    setTimeout(function () {
+      res.end(req.url)
+    }, 10)
```

Comment:
please avoid the timeout, it would make the tests even more brittle

Reply:
removed, was not needed, came with copy paste of another test